docs/integrations/datadog-rum/installation.md

```diff
@@ -26,7 +26,48 @@ app:
   #   sessionReplaySampleRate: 0
 ```
 
-If your [`app-config.yaml`](https://github.com/backstage/backstage/blob/e0506af8fc54074a160fb91c83d6cae8172d3bb3/app-config.yaml#L5) file does not have this configuration, you may have to adjust your [`packages/app/public/index.html`](https://github.com/backstage/backstage/blob/e0506af8fc54074a160fb91c83d6cae8172d3bb3/packages/app/public/index.html#L69) to include the Datadog RUM `init()` section manually.
+If your [`app-config.yaml`](https://github.com/backstage/backstage/blob/e0506af8fc54074a160fb91c83d6cae8172d3bb3/app-config.yaml#L5) file does not have this configuration, you may have to adjust your [`packages/app/public/index.html`](https://github.com/backstage/backstage/blob/e0506af8fc54074a160fb91c83d6cae8172d3bb3/packages/app/public/index.html#L69) to include the Datadog RUM `init()` section manually. If you a Datadog not publishing issue copy and paste this section in to your `packages/app/public/index.html`
+
+```
+<% if (config.has('app.datadogRum')) { %>
+ <script>
+   (function (h, o, u, n, d) {
+     h = h[d] = h[d] || {
+       q: [],
+       onReady: function (c) {
+         h.q.push(c);
+       },
+     };
+     d = o.createElement(u);
+     d.async = 1;
+     d.src = n;
+     n = o.getElementsByTagName(u)[0];
+     n.parentNode.insertBefore(d, n);
+   })(
+     window,
+     document,
+     'script',
+     'https://www.datadoghq-browser-agent.com/datadog-rum-v3.js',
+     'DD_RUM',
+   );
+   DD_RUM.onReady(function () {
+     DD_RUM.init({
+       clientToken: '<%= config.getString("app.datadogRum.clientToken") %>',
+       applicationId:
+         '<%= config.getString("app.datadogRum.applicationId") %>',
+       site: '<%= config.getOptionalString("app.datadogRum.site") || "datadoghq.com" %>',
+       service: 'backstage',
+       env: '<%= config.getString("app.datadogRum.env") %>',
+       sampleRate:
+         '<%= config.getOptionalNumber("app.datadogRum.sessionSampleRate") || 100 %>',
+       sessionReplaySampleRate:
+         '<%= config.getOptionalNumber("app.datadogRum.sessionReplaySampleRate") || 0 %>',
+       trackInteractions: true,
+     });
+   });
+ </script>
+ <% } %>
+```
```

Comment:
Maybe format this as a "note" instead

```suggestion
If your [`app-config.yaml`](https://github.com/backstage/backstage/blob/e0506af8fc54074a160fb91c83d6cae8172d3bb3/app-config.yaml#L5) file does not have this configuration, you may have to adjust your [`packages/app/public/index.html`](https://github.com/backstage/backstage/blob/e0506af8fc54074a160fb91c83d6cae8172d3bb3/packages/app/public/index.html#L69) to include the Datadog RUM `init()` section manually. 

:::note
If you encounter a Datadog not publishing issue, copy and paste this section into your `packages/app/public/index.html`:
```html
<% if (config.has('app.datadogRum')) { %>
 <script>
   (function (h, o, u, n, d) {
     h = h[d] = h[d] || {
       q: [],
       onReady: function (c) {
         h.q.push(c);
       },
     };
     d = o.createElement(u);
     d.async = 1;
     d.src = n;
     n = o.getElementsByTagName(u)[0];
     n.parentNode.insertBefore(d, n);
   })(
     window,
     document,
     'script',
     'https://www.datadoghq-browser-agent.com/datadog-rum-v3.js',
     'DD_RUM',
   );
   DD_RUM.onReady(function () {
     DD_RUM.init({
       clientToken: '<%= config.getString("app.datadogRum.clientToken") %>',
       applicationId:
         '<%= config.getString("app.datadogRum.applicationId") %>',
       site: '<%= config.getOptionalString("app.datadogRum.site") || "datadoghq.com" %>',
       service: 'backstage',
       env: '<%= config.getString("app.datadogRum.env") %>',
       sampleRate:
         '<%= config.getOptionalNumber("app.datadogRum.sessionSampleRate") || 100 %>',
       sessionReplaySampleRate:
         '<%= config.getOptionalNumber("app.datadogRum.sessionReplaySampleRate") || 0 %>',
       trackInteractions: true,
     });
   });
 </script>
 <% } %>
```
:::
```